quest/api/datasets.py

```diff
@@ -415,16 +417,16 @@ def visualize_dataset(dataset, update_cache=False, **kwargs):
     if path is None:
             raise ValueError('No dataset file found')
 
-    if file_format not in list_plugins('io'):
+    if file_format not in list_plugins(static.PluginType.IO):
             raise ValueError('No reader available for: %s' % file_format)
 
-    io = load_plugins('io', file_format)[file_format]
+    io = load_plugins(static.PluginType.IO, file_format)[file_format]
 
     title = m.get('display_name')
     if title is None:
         title = dataset
 
-    if 'timeseries' in file_format:
+    if static.DataType.TIMESERIES in file_format:
```

Comment:
This shouldn't be `DataType` since `datatype` and `file_format` are distinct in the database. We may need to create a new static enum for `file_format`.